src/index.ts

```diff
@@ -36,25 +39,20 @@ const setExpiration = (key: string, expirationTime: string): void => {
   const allowedFormats = Object.keys(timeFormats);
   const timeKey = expirationTime[expirationTime.length - 1];
   const time = Number(expirationTime.replace(timeKey, ""));
-  if (expirationTime.length < 2 || !allowedFormats.some((char) => timeKey === char) || isNaN(time))
-    return console.warn("Localit: provide a valid expiration time format (e.g. '20h', '160s', '15d'). Your expiration date hasn't been saved.");
+  if (expirationTime.length < 2 || !allowedFormats.some((char) => timeKey === char) || isNaN(time)){
+    console.warn("Localit: provide a valid expiration time format (e.g. '20h', '160s', '15d'). Your expiration date hasn't been saved.");
+    return;
+  }
 
-  store.setItem(getExpirationKey(key), JSON.stringify(timeFormats[timeKey](time)));
+  return timeFormats[timeKey](time);
 };
 
-/**
- * @param key - the key to check if it has an expiration date
- * @return whether or not there is an expiration date for the given key
- */
-const hasExpirationDate = (key: string): boolean => store.getItem(getExpirationKey(key)) !== null;
-
 /**
  * @param key - the key to check if it has expired
  * @return whether or not there is an expiration date for the given key
  */
-const hasExpired = (key: string): boolean => {
-  const expirationDate: string = JSON.parse(store.getItem(getExpirationKey(key)));
-  return new Date() > new Date(expirationDate);
+const hasExpired = (time: number): boolean => {
```

Comment:
Make this a one-liner for the swag points!

Reply:
I want my swag point